DigitalLearningSolutions.Web.Tests/ViewModels/MyAccount/MyAccountViewModelTests.cs

```diff
@@ -123,5 +131,83 @@ public void MyAccountViewModel_CustomFields_ShouldBePopulated()
                 returnedModel.CustomFields[1].Mandatory.Should().BeFalse();
             }
         }
+
+        [Test]
+        public void MyAccountViewModel_where_user_has_not_been_asked_for_prn_says_not_yet_provided()
+        {
+            // Given
+            var delegateUser = UserTestHelper.GetDefaultDelegateUser(
+                hasBeenPromptedForPrn: false,
+                professionalRegistrationNumber: "123456"

```

Comment:
I think it makes more sense to test the case where `professionalRegistrationNumber` is null, because that's what we expect to see in production, and it's entirely possible that `professionalRegistrationNumber: null` would override `hasBeenPromptedForPrn: false`.